app/assets/javascripts/kassi.js

```diff
@@ -337,9 +337,8 @@ function updateSellerGetsValue(priceInputSelector, displayTargetSelector, curren
     }
 
     $display.text(
-      [ST.paymentMath.displayMoney(Math.max(0, displaySum)),
-       $currency.val()]
```

Comment:
Actually, I'm going to leave it as is. It makes sense to keep all the jQuery element creations together.